doc/classes/FuncRef.xml

```diff
@@ -24,7 +25,7 @@
 			<argument index="0" name="name" type="String">
 			</argument>
 			<description>
-				Set the name of the function to call on the object, without parentheses or any parameters.
+				Sets the name of the function to call on the object, without parentheses or any parameters.
```

Comment:
Ref writing guidelines: "Don’t repeat the method’s name. It already states it sets":

    The name of the referenced function.